arrows/serialize/json/tests/test_load_save.cxx

```diff
@@ -226,6 +226,12 @@ kwiver::vital::metadata create_meta_collection()
     meta.add( item );
   }
 
+  {
+    const auto& info = traits.find( kwiver::vital::VIDEO_KEY_FRAME );
```

Comment:
This is the line Travis is choking on. If you get it to build and repush, I will make sure the other CI is run as well.

Reply:
@russelldj It probably needs to be `kwiver::vital::VITAL_VIDEO_KEY_FRAME`.  The macros in metadata_tags.h prepend `VITAL_` to each of those identifiers when making the enum.

Reply:
Has this been fixed?

Reply:
I've been having issues getting GTests to build locally and haven't had much time to dig into it. But Matt L. suggested I push up this change.